app/utils/services/token.service.js

```diff
@@ -23,5 +26,10 @@
             $log.debug(`==> Save token: ${token}`);
             return $localStorage.token = token;
         }
+
+        function clean() {
+            $localStorage.token = null;
+            $window.location.pathname = '/';
```

Comment:
Please, use redirect method in the controller, not in the clean() method. For example:
`TokenService.clean();`
`$window.location.pathname = '/'`